Geometry/ForwardCommonData/data/zdcmaterials/2021/v1/zdcmaterials.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<DDDefinition>
+ <MaterialSection label="zdcmaterials.xml">
+  <CompositeMaterial name="Air" density="1.29*mg/cm3" symbol=" " method="mixture by weight">
+   <MaterialFraction fraction="0.7">
+    <rMaterial name="materials:Nitrogen"/>
+   </MaterialFraction>
+   <MaterialFraction fraction="0.3">
+    <rMaterial name="materials:Oxygen"/>
+   </MaterialFraction>
+  </CompositeMaterial>o
```

Comment:
Looks like there is an "o" after `</CompositeMaterial>`. Is that valid?

Reply:
It did not create an error when i tested it. I shall fix it later. it works with that